cypress/integration/tests/cart.js

```diff
@@ -0,0 +1,32 @@
+import CartPopUp from '../../page-objects/cartPopUp';
+import products from '../../fixtures/products.json';
+import { pickTargetCategory, pickTargetProducts } from '../../utils/helpers';
+import ShopPage from '../../page-objects/shopPage';
+
+describe('Cart tests', () => {
+  let productCategory,
+    targetProducts;
```

Comment:
okay, lets view on 3 ways of writing this vars declaration thing

1. Recommended
```
let something;
let something1;
let something2;
```
2. What I am doing on my project
```
let something, something1, something2;
```
3. yours
```
let something,
 something1,
 something2;
```
Stick with method 1 - the best readability.

Reply:
done